File: android/expoview/src/main/java/versioned/host/exp/exponent/ReactUnthemedRootView.java
// Copyright 2015-present 650 Industries. All rights reserved.

package versioned.host.exp.exponent;

import android.content.Context;
import android.view.ContextThemeWrapper;

import versioned.host.exp.exponent.modules.api.components.gesturehandler.react.RNGestureHandlerEnabledRootView;
import host.exp.expoview.R;

public class ReactUnthemedRootView extends RNGestureHandlerEnabledRootView {
  public ReactUnthemedRootView(Context context) {
    super(new ContextThemeWrapper(
        context,
        R.style.Theme_Exponent_None
    ));
  }
}

File: android/expoview/src/main/java/versioned/host/exp/exponent/ReactUnthemedRootView.kt
// Copyright 2015-present 650 Industries. All rights reserved.
package versioned.host.exp.exponent

import android.content.Context
import android.view.ContextThemeWrapper
import host.exp.expoview.R
import com.facebook.react.ReactRootView

class ReactUnthemedRootView(context: Context?) : ReactRootView(
  ContextThemeWrapper(
    context,
    R.style.Theme_Exponent_None
  )
)
